### ISSUE TITLE
Permissions on atlantis_iceland

### ISSUE THREAD

**Issue by pfrater** [mareframe/mfdb#50]
I'm having trouble gaining access to the atlantis_iceland schema of the mfdb located at mfdb.rhi.hi.is. Here is the R code I used:

`mdb <- mfdb("Atlantis-Iceland", db_params = list(host = "mfdb.rhi.hi.is"))`

and the error message I receive

```
Error in postgresqlExecStatement(conn, statement, ...) : 
  RS-DBI driver: (could not Retrieve the result : ERROR:  permission denied for schema atlantis_iceland
LINE 2: CREATE TABLE atlantis_iceland.mfdb_schema (
                     ^
)
```

I should have permissions for this schema. When I run the following from the server terminal my username is listed as having permission.

`psql mf`
`\l`

Also, I know it's not an incorrect username/password because I can create a new schema.

`mdb <- mfdb('zebrafish', db_params=list(host = 'mfdb.rhi.hi.is'))`
`mfdb('zebrafish', db_params = list(host = 'mfdb.rhi.hi.is'), destroy_schema=T)`

**Comment by lentinj** [comment]
Do you need a ``username = "pfrater"`` in ``db_params``?

The owner of atlantis_iceland was me, I've changed it to you.

**Comment by pfrater** [comment]
No, adding the `username = 'pfrater'` in `db_params` throws an "unused parameter" warning.

I re-installed mfdb from github just to make sure I had the most current version, now from my local desktop I am just getting the `Could not find a local mf database` error. However, the original command also fails from the Rstudio web app (i.e. https://mfdb.rhi.hi.is/rstudio).

I am getting the "Iceland" schema to work, but I still get the following error for "Atlantis-Iceland"

```
mdb <- mfdb('Atlantis_Iceland', db_params = list(host = 'mfdb.rhi.hi.is'))
Username: pfrater
Error in postgresqlExecStatement(conn, statement, ...) : 
  RS-DBI driver: (could not Retrieve the result : ERROR:  must be owner of function weighted_mean_accum
)
```

**Comment by lentinj** [comment]
Hrm, that any better?

If not I should probably just delete what's there for atlantis_iceland, I'm guessing it's not that useful.

**Comment by pfrater** [comment]
Yes! Now I can get a connection on https://mfdb.rhi.hi.is/rstudio.

From my local machine I still get the following error: 

```
mdb <- mfdb('Atlantis-Iceland', db_params = list(host = 'mfdb.rhi.hi.is'))
Error in mfdb("Atlantis-Iceland", db_params = list(host = "mfdb.rhi.hi.is")) : 
  Could not find a local mf database
```

Do I have to do something to something different when on a local machine?

**Comment by lentinj** [comment]
Is it prompting you for a username / password?

**Comment by pfrater** [comment]
No, it just tells me that it "Could not find a local mf database"

**Comment by pfrater** [comment]
Well I've figured out the issue on my local machine. Somehow my `mfdb` got reverted to version 1.0.0. I've been reading Hadley Wickham's [Advanced R book](http://adv-r.had.co.nz/) and was doing the exercises on environments and think I messed something up there (my `.libPaths()` also got reverted to default). So, this now works well on my local machine. Still gives an error when I add `username = pfrater` into the db_params argument, but it prompts me for one if I don't.
